docs/reference/team-processes/development-process.md

```diff
@@ -142,6 +142,40 @@ If a branch will survive beyond a couple of days its important to have a bit of
 * You should rebase against main often to avoid a giant headache when you eventually merge
 * Consider creating new components instead of editing old ones if you have extensive enough changes.  This way it's a very small change to flip from one to the other instead of trying to merge.
 
+## Pull requests
+
+At the time of writing, FxA engineers push their branches to `mozilla/fxa` directly; we have [an issue we closed as a won't fix][issue-forked-prs] detailing why we currently don't use forks. Try to delete your stale or already merged branches on a regular basis.
```

Comment:
I'm going to file an issue for this but we need to update our CONTRIBUTING.md file with note about this, as well as a few other pieces that are out of date😬